### PR TITLE
Changed Mashov to configYAML branch

### PR DESCRIPTION
**Pull request by UdiMizrachi**
Also added option for /sign_all for multiple signing platforms

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     build: 
       context: .
       dockerfile: ./Dockerfile
-    container_name: dockerbot
+    container_name: dockerbot-test
```
Comment (AdamRussak) [inline docker-compose.yml]: just remove the -test

Comment (AdamRussak) [reply docker-compose.yml]: `container_name: dockerbo`

File: dockerbot.py [hunk 2]
```diff
@@ -1,52 +1,252 @@
...
+
 bot = telepot.Bot(os.getenv('API_KEY'))
 
 MessageLoop(bot, handle).run_as_thread()
 
 logger.info('I am listening...')
- 
+
 while 1:
     time.sleep(10)
```
Comment (AdamRussak) [inline dockerbot.py:249]:
```suggestion
while 1:
    time.sleep(10)
    #check Conifg is existed
    original = r'/etc/config.yml'
    target = r'/opt/config/config.yml'
    if os.path.isfile(target):
        continue
    else:
        shutil.copyfile(original, target)
        logger.info("Copyed Config to /opt/config")
```

File: dockerbot.py [hunk 3]
```diff
@@ -1,52 +1,260 @@
...
+             logger.exception(f"[{message_id}] Failed to handle command. Msg: {command}")
+             bot.sendMessage(chat_id, f"ERROR: {str(ex)}")
+
     msg = f"Done message handling: {command}"
     logger.info(f"[{message_id}] {msg}")
 
 
+    if command == '/sign_all':
+        if list['auto_sign']['edu'] == 1:
+            Image = '/opt/dockerbot/images/edu_approval.png'
```
Comment (AdamRussak) [inline dockerbot.py:159]: the Image is not in the Repo

Comment (t0mer) [reply dockerbot.py:159]: There is an image foreach healthstatment.
if the sign sucsseded the image than will be created and send

File: docker-compose.yml (hunk 1 above)
Comment (UdiMizrachi) [reply docker-compose.yml]: Done, you don`t want also to remove  - - /home/adam/test-volume:/opt/config ?

File: dockerbot.py (hunk 2 above)
Comment (AdamRussak) [reply dockerbot.py:249]:
```suggestion
while 1:
    time.sleep(10)
    #check Conifg is existed
    original = r'/etc/config.yml'
    target = r'/opt/dockerbot/config/config.yml'
    if os.path.isfile(target):
        continue
    else:
        shutil.copyfile(original, target)
        logger.error("Recoverd Config to /opt/dockerbot/config/")
```

**Pull request merged**